src/style.css

```diff
@@ -0,0 +1,897 @@
+    url("./assets/fonts/Roboto-Regular.ttf") format('truetype');
+} 
+
+@font-face{
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 300;
+  src: 
+    local("Roboto-Light"),
+    url("./assets/fonts/Roboto-Light.ttf") format('truetype');
+} 
+
+@font-face{
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 500;
+  src: 
+    local("Roboto-Medium"),
+    url("./assets/fonts/Roboto-Medium.ttf") format('truetype');
+} 
+@font-face{
+  font-family: "Comfortaa";
+  font-style: normal;
+  font-weight: 400;
+  src: 
+    local("Comfortaa"),
+    url("./assets/fonts/Comfortaa-Regular.ttf") format('truetype');
+} 
+
+@font-face{
+  font-family: "Comfortaa";
+  font-style: normal;
+  font-weight: 300;
+  src: 
+    local("Comfortaa"),
+    url("./assets/fonts/Comfortaa-Light.ttf") format('truetype');
+} 
+
+@font-face{
+  font-family: "Raleway";
+  font-style: normal;
+  font-weight: 500;
+  src: 
+    local("Raleway"),
+    url("./assets/fonts/Raleway-Medium.ttf") format('truetype');
+} 
+
+:root{
+  --color-primary: rgba(220, 240, 250, 1);
+  --color-primary-light:rgba(220, 240, 250, 0.5);
+  --color-primary-dark: rgba(150, 180, 200, 1);
+  --color-overlay: rgba(130, 180, 205, 1);
+  --color-secondary: rgb(112, 197, 228);
+  --color-accent: rgba(255, 215, 112, 1);
+
+  --btn-shadow: 0px 2px 10px rgba(167, 176, 182, 0.2);
+  --btn-hover-shadow: 1px 5px 15px rgba(87, 93, 97, 0.15);
+
+  --hard-shadow: 3px 3px 8px rgba(0, 0, 0, 0.4);
+  --overlay-shadow: 0px -5px 3px rgba(0, 0, 0, 0.3);
+  --card-shadow: 3px 3px 10px rgba(0, 0, 0, 0.2), 0px 0px 40px rgba(0, 0, 0, 0.2);
+
+  --color-contrast-text:  #000 ;
+  --color-light-text: #fff;
+  --color-regular-text: #333;
+  --color-overlay-text: rgb(39, 79, 116);
+
+  --font-regular: "Roboto";
+  --font-light:"Roboto", 300;
+  --font-bold:"Roboto", 500;
+  --font-pretty: "Comfortaa";
+
+  --text-large: 2rem;
+  --text-medium: 1.5rem;
+  --text-small: 1rem;
+
+  --preview-width: 250px; 
+}
+
+@media(prefers-color-scheme: dark){
+  :root{
+    --color-primary: #376F89;
+    --color-primary-light:rgba(29, 183, 237, 0.5);
+    --color-primary-dark: rgba(150, 180, 200, 1);
+    --color-overlay: rgba(130, 180, 205, 1);
+    --color-secondary: rgba(29, 183, 237, 1);
+    --color-accent: rgba(255, 215, 112, 1);
+    --light-shadow: 0 0 8px rgba(0, 0, 0, 0.1);  
+    --card-shadow: 5px 5px 20px rgba(255, 255, 255, 0.4);
+
+    --color-contrast-text:  #fff ;
+    --color-regular-text: #fff;
+    --color-overlay-text: rgb(227, 242, 255);
+  }
+  body{
+    background-color: #222;
+  }
+  
+}
+@media only screen and (max-width: 650px){
+  :root{
+    font-size: 14px;
+  }
+}
+
+/*global*/
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  line-height: 1.4em;
+  font-family: var(--font-regular);
+
+} 
+
+a{
+  text-decoration: none
+} 
+
+section{
+  width: 100%;
+  margin-bottom: 4rem;  
+  color: var(--color-regular-text);
+}
+
+.app-container{
+  height: 100vh;
+  width: 100vw;
```

Comment:
Normally, a block component will take all available space without this line. With it, on the other hand, some extra bugs may occur: different browsers treat this value differently if there is a vertical scrollbar on the page. For example Chrome adds scrollbar width to 100vw value and we have overflow which adds another scrollbar (horizontal), which is very bad. So there is a general advice to not rely on 100vw value at all.

```suggestion
```

Read more: https://ishadeed.com/article/viewport-units/